lms/djangoapps/support/tests/test_views.py

```diff
@@ -1189,3 +1189,177 @@ def test_fbe_disabled_response(self):
         )
         data = json.loads(response.content.decode('utf-8'))
         assert data == {}
+
+
+@ddt.ddt
+class LinkProgramEnrollmentSupportAPIViewTests(SupportViewTestCase):
+    """
+    Tests for the link_program_enrollments support view.
+    """
+
+    def setUp(self):
+        """Make the user support staff. """
+        super().setUp()
+        self.url = reverse("support:link_program_enrollments_details")
```

Comment:
you can make this class level variable. url will remain the same, no matter what.